TUTORIAL.rst

```diff
@@ -0,0 +1,187 @@
+
+        * Run the ``pdfmorph --help`` command and read over the
+          info on that page for a brief overview of some of what we will
+          explore in this tutorial.
+
+	3. Using the ``mkdir`` command, create a directory where you'll
+        store the tutorial PDF files and use the ``cd`` command to change
+        into that directory. You can download the tutorial files
+        :download:`here <tests/testdata/tutorialData.zip`. Then, ``cd``
+        into the ``tutorialData`` directory.
+
+        * The files in this dataset were collected by Soham Banerjee
+          at Brookhaven National Laboratory in Upton, New York.
+
+        * The files are PDF data collected on Iridium Telluride with
+          20% Rhodium Doping (IrRhTe2) with the first file (01) collected
+          at 10K and the last (44) at 300K. The samples increase in
+          temperature as their numbers increase. The "C" in their names
+          indicates that they have undergone cooling.
+
+        * Note that these files have the ``.gr`` extension, which
+          indicates that they are measured PDFs. The ``.cgr`` file
+          extension indicates that a file is a calculated PDF, sich as
+          those generated by the `PDFgui <https://www.diffpy.org/products/pdfgui.html>`_
+          program.
+
+    4. First, we will run the PDFmorph application without any morphing
+       and only using one PDF. Type the following command into your
+       Command Line ::
+
+            pdfmorph darkSub_rh20_C_01.gr darkSub_rh20_C_01.gr
+
+        This should produce two PDF curves which are congruent, resulting
+        in a flat green line underneath them.
+
+    5. Now, we will see PDFmorph run with two different PDFs and no
+       morphing. Type the following command into your command line ::
+
+            pdfmorph darkSub_rh20_C_01.gr darkSub_rh20_C_44.gr
+
+       Without morphing, the difference Rw = 0.407. This indicates that
+       the two PDFs vary drastically.
+
+            * While running the ``pdfmorph`` command, it is important
+              to remember that the first PDF file argument you provide
+              (in this case, ``darkSub_rh20_C_01.gr``) is the PDF which
+              will get morphed, while the second PDF file argument you
+              provide (here, ``darkSub_rh20_C_44.gr``) is the PDF which
+              acts as the model and does not get morphed. Hereinafter,
+              we will refer to the first PDF argument as the "objective"
+              and the second as the "reference", as the PDFmorph display
+              does.
+
+    6. Now, we will start the morphing process, which requires us to
+       provide initial guesses for our scaling factor, Gaussian smear,
+       and stretch, separately. We will start with the scaling factor.
+       Begin by typing the command ::
+
+            pdfmorph --scale=2 -a darkSub_rh20_C_01.gr darkSub_rh20_C_44.gr
+
+       Now, the difference Rw = 1.457, a significant increase from our
```

Comment:
the tutorial will be better if you insert figures showing the datasets plotted on top of each other (i.e., the output from PDFmorph given the inputs you gave.  This will be sanity check for the user that they are gietting what they should be getting